core/src/main/java/org/apache/accumulo/core/classloader/ClassLoaderUtil.java

```diff
@@ -82,6 +82,14 @@ public static ClassLoader getClassLoader(String context) {
     }
   }
 
+  public static boolean isValidContext(String context) {
+    if (context != null && !context.isEmpty()) {
+      return FACTORY.isValid(context);
```

Comment:
I think this is the only line needed here.

Reply:
This is OBE, the context value is no longer checked in this method. It has been pushed into the factory.